### PR TITLE
Choose template folder for report

### PR DESCRIPTION
**Pull request by cosletmihai**
- update pictures
- remove currently not required folder (`rdf_diff_template_generator`)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -35,6 +35,27 @@ To run the tests:
...
 ```
+## Configure diff report template 
+The default diff report template resides in [resources/eds_templates/diff_report](resources/eds_templates/diff_report). 
+
+To configure your own template you can copy the default report template and adjust it to your needs. Read more about the required structure of the template on the [eds4jinja2](https://github.com/meaningfy-ws/eds4jinja2) documentation page.
+ 
+### Use the custom template
+After you have your custom template, run the `make` command, indicating the location of your template through the `location` variable.
+```bash
+make location=<location to template> set-report-template
```
Comment (costezki) [inline README.md:46]: great idea!

File: README.md [hunk 2]
```diff
@@ -35,6 +35,27 @@ To run the tests:
 make fuseki-create-test-dbs
 make test
 ```
+## Configure diff report template
```
Comment (costezki) [inline README.md:38]: well done: adding documentation.

File: Makefile [hunk 3]
```diff
@@ -17,6 +17,7 @@ install:
 
 build-services:
 	@ echo -e '$(BUILD_PRINT)Building the RDF Differ micro-services'
+	@ docker volume create rdf-differ-template
```
Comment (costezki) [inline Makefile:20]: I would not create the volume here; this shall be handled by another make target.
just delete this line and add the dependency to the make target below.

File: Makefile [hunk 4]
```diff
@@ -33,6 +34,7 @@ stop-services:
 #-----------------------------------------------------------------------------
 build-test-fuseki:
 	@ echo -e '$(BUILD_PRINT)Building the Fuseki service'
+	@ docker volume create rdf-differ-template
```
Comment (costezki) [inline Makefile:37]: Fuseki does not need this volume

File: .github/workflows/test.yml [hunk 5]
```diff
@@ -14,6 +14,7 @@ jobs:
     env:
       OS: ${{ matrix.os }}
       PYTHON: '3.8'
+      RDF_DIFFER_TEMPLATE_LOCATION: /usr/src/app/template
```
Comment (costezki) [inline .github/workflows/test.yml:17]: do we need it here, or as a part of .env file?

File: rdf_differ/entrypoints/api/handlers.py [hunk 6]
```diff
@@ -134,7 +134,7 @@ def get_report(dataset_id: str) -> tuple:
 
     try:
         with tempfile.TemporaryDirectory() as temp_dir:
-            template_location = Path(__file__).parents[3] / 'resources/eds_templates/diff_report'
+            template_location = config.RDF_DIFFER_REPORT_TEMPLATE_LOCATION
```
Comment (costezki) [inline rdf_differ/entrypoints/api/handlers.py:137]: that's it!

**Review by costezki — APPROVED** [review]
well done;
update only the Makefile

**Review by costezki — APPROVED** [review]

File: Makefile (hunk 4 above)
Comment (cosletmihai) [reply Makefile:37]: fuseki doesn't, but `docker-compose` does, and it won't start without having all defined external volumes created in the `volumes` in the `docker-compose.yml`

**Pull request merged**